packages/wallet/src/cip30.ts

```diff
@@ -465,10 +465,10 @@ const baseCip30WalletApi = (
   },
   signTx: async ({ sender }: SenderContext, tx: Cbor, partialSign?: Boolean): Promise<Cbor> => {
     const scope = new ManagedFreeableScope();
-    logger.debug('signTx');
-    const txDecoded = Serialization.Transaction.fromCbor(Serialization.TxCBOR(tx));
+    logger.debug('signTx', tx);
```

Comment:
@rhyslbw , @AngelCastilloB . Is there any issue if we log the cbor we're about to sign? It has been very helpful in my debugging.

Reply:
No issues, this is suggested